images/build/benv.sh

```diff
@@ -217,6 +217,8 @@ benv-resolve() {
     # successfully compiled and linked.
     export LIBRARY_PATH="$platformDir/lib:$LIBRARY_PATH"
     export LD_LIBRARY_PATH="$platformDir/lib:$LD_LIBRARY_PATH"
+    export C_INCLUDE_PATH="$platformDir/include/python$majorAndMinorParts"
```

Comment:
Would you mind including a comment above these two environment variables that elaborate on their purpose, similar to what we're doing above with `LIBRARY_PATH` and `LD_LIBRARY_PATH`?

Reply:
Yup! Updated